app/models/manageiq/providers/vmware/infra_manager/cluster.rb

```diff
@@ -31,8 +31,8 @@ def register_host(host)
       end
 
       host.ems_ref                = host_mor
-      host.ems_ref_obj            = host_mor
```

Comment:
Is this deletion on purpose?

Reply:
Yes, we no longer allow setting this property after the column was removed and this was missed so it was failing the new test